### PR TITLE
add enum methods

### PR DESCRIPTION
**Pull request by WangechiMuchiri**

### CODE REVIEW

File: CliTool/Commands/EntityTypeCommand.cs [hunk 1]
```diff
@@ -20,6 +22,39 @@ public EntityTypeCommand()
...
+
+                return;
+            }
+
+
+            CsdlXPath csdlXPath = new CsdlXPath(filePath);
+            EntityType entityType = csdlXPath.GetEntityType(name);
+
+            Console.WriteLine("***Entity Type***");
+            Console.WriteLine($"{entityType.Name}\n");
```
Comment (KenitoInc) [inline CliTool/Commands/EntityTypeCommand.cs:55]: Can we have the name and key in one line. Something like

`EntityType Name: Person, Key: UserName`

File: CliTool/Commands/EnumMembersCommand.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+            if (name == null)
+            {
+                Console.WriteLine($"Ensure you add the option --name");
+
+                return;
+            }
+
+            CsdlXPath csdlXPath = new CsdlXPath(filePath);
+            /*List<EdmMember> enumMembers = csdlXPath.GetEnumMembers(name);
```
Comment (KenitoInc) [inline CliTool/Commands/EnumMembersCommand.cs:54]: Why is it commented?

File: CsdlXPathLib/EdmTypes/EdmMember.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CsdlXPathLib.EdmTypes
+{
+    public class EdmMember
```
Comment (KenitoInc) [inline CsdlXPathLib/EdmTypes/EdmMember.cs:9]:
```suggestion
    public class EnumMember
```

File: CsdlXPathLib/CsdlXPath.cs [hunk 4]
```diff
@@ -80,20 +81,30 @@ public List<EntityType> GetEntityTypes()
...
-            results.MoveNext();
+            nodes.MoveNext();
             string name = string.Empty;
-
-            if (results.Current.MoveToAttribute(XmlConstants.Name, ""))
-            {
-                name = results.Current.Value;
+            string key = string.Empty;
+            if (nodes.Current.MoveToAttribute(XmlConstants.Name, ""))
+             {
```
Comment (KenitoInc) [inline CsdlXPathLib/CsdlXPath.cs:90]:
```suggestion
            {
```

File: CsdlXPathLib/EdmTypes/EnumType.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CsdlXPathLib.EdmTypes
+{
+    public class EnumType
+    {
+        public string Key { get; set; }
+        public string Name { get; set; }
+        public List<EdmMember> Members { get; set; }
```
Comment (KenitoInc) [inline CsdlXPathLib/EdmTypes/EnumType.cs:13]:
```suggestion
        public List<EnumMember> Members { get; set; }
```

File: CliTool/Commands/EnumMembersCommand.cs (hunk 2 above)
Comment (WangechiMuchiri) [reply CliTool/Commands/EnumMembersCommand.cs]: I have now added its method

**Pull request merged**